pyproject.toml

```diff
@@ -13,34 +19,21 @@ filterwarnings = [
     "error:::libqtile",
 ]
 
-[tool.setuptools-git-versioning]
-enabled = true
+[tool.setuptools_scm]
 
 [tool.setuptools.dynamic]
 readme = {file = "README.rst"}
 
+[tool.setuptools.packages.find]
+include = ["libqtile*"]
+
 [project]
 name = "qtile"
 description = "A pure-Python tiling window manager."
-dynamic = ["version", "readme"]
-license = {text = "MIT"}
-classifiers = [
-    "Intended Audience :: End Users/Desktop",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: POSIX :: BSD :: FreeBSD",
-    "Operating System :: POSIX :: Linux",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: Implementation :: PyPy",
-    "Topic :: Desktop Environment :: Window Managers",
```

Comment:
Don't we want to keep most of this metadata? In particular, this is what it uses to render the sidebar here IIUC: https://pypi.org/project/qtile/

(You could probably drop all the classifiers, but at least the license and links to docs/homepage/code seem useful)

Reply:
Yes - I'm not sure how that got deleted here!